Exoft.Gamification.Api.Data/Repositories/UserRepository.cs

```diff
@@ -18,26 +17,35 @@ public UserRepository(UsersDbContext context) : base(context)
 
         public async Task<User> GetUserAsync(Guid Id)
         {
-            var user = await DbSet
-                .Include(s => s.Roles)
-                    .ThenInclude(s => s.Role)
-                .Include(s => s.Achievements)
-                    .ThenInclude(s => s.Achievement)
-                .FirstOrDefaultAsync(i => i.Id == Id);
+            var user = await IncludeAll().FirstOrDefaultAsync(i => i.Id == Id);
 
             return user;
         }
 
         public async Task<User> GetUserAsync(string userName)
         {
-            var user = await DbSet
+            var user = await IncludeAll().FirstOrDefaultAsync(i => i.UserName == userName);
+
+            return user;
+        }
+
+        public async Task<ICollection<User>> GetUsersAsync()
+        {
+            var list = new List<User>();
```

Comment:
Emmm.... I would not say anything about code quality here and knowledge of linq...
So please remove this  method for now... we will get back to it once you start working on API for some entity